rdt/performance/performance.py

```diff
@@ -13,24 +13,13 @@
         'missing_value_replacement': -1,
         'model_missing_values': True
     },
-    'DatetimeTransformer': {
-        'missing_value_replacement': 'mean',
+    'UnixTimestampEncoder': {
```

Comment:
Unrelated change: I think we just forgot to remove these when we renamed the transformers.